drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -83,6 +83,11 @@ def auto_exit(enrollment, most_recent_contact)
       )
       assessment.build_form_processor(exit: exit_record)
       assessment.save!
+
+      # Release the unit that was assigned to this Enrollment (if applicable)
+      enrollment.release_unit!(exit_date, user: system_user)
```

Comment:
This queues up a bunch of jobs, one per exited enrollment (if AC HMIS Extension is enabled). That could become problematic. But I think OK because there shouldn't be very many enrollments exited per day

Reply:
I think this will help https://github.com/greenriver/hmis-warehouse/pull/4025/commits/c38b206b8c8dd39a5456d2c4598477b86c95bf57

Reply:
Does `HmisExternalApis::AcHmis::UpdateUnitAvailabilityJob` update all units?  I think c38b206 should be good enough if it does.